app/commands/solution/mark_as_out_of_date_in_index.rb

```diff
@@ -0,0 +1,32 @@
+class Solution::MarkAsOutOfDateInIndex
+  include Mandate
+
+  initialize_with :exercise
+
+  def call
+    client.update_by_query(index: 'solutions', body: {
+      script: {
+        source: 'ctx._source.out_of_date = false'
+      },
+      query: {
+        bool: {
+          must: [
+            { term: { 'exercise_id': exercise.id } },
+            { term: { 'out_of_date': true } }
```

Comment:
Does this not only change things that are `out_of_date = true` and change them to `out_of_date` = false? Did you mean to call the field "up_to_date" or is the logic wrong, or am I misreading it?